packages/table/src/table.tsx

```diff
@@ -542,6 +542,7 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
             this.bodyElement,
             this.grid,
         );
+        this.locator = locator;
 
         const viewportRect = locator.getViewportRect();
         this.setState({ locator, viewportRect });
```

Comment:
I was worried about the child stuff not rerendering at all, without an update to the state. 

But since the state's being updated anyways, you're right, this is fine. My bad.